src/borg/archiver/config_cmd.py

```diff
@@ -146,7 +146,7 @@ def build_parser_config(self, subparsers, common_parser, mid_common_parser):
         For security reasons, this command only works on local repositories.
 
         To delete a config value entirely, use ``--delete``. To list the values
-        of the configuration file or the default values, use ``--list``.  To get and existing
```

Comment:
Thanks for the PR!

As documented in the first lines of borg-config.1 and config.rst.inc, these 2 files are generated.

Thus, config_cmd.py is the only file that needs to be changed to fix this typo. Can you remove the other files from this PR, please?

Reply:
Done!